inputFiles/hydraulicFracturing/heterogeneousInSitu_base.xml

```diff
@@ -108,10 +108,10 @@
       target="/Solvers/hydrofracture"/>
 
     <!-- Watch the wall-clock, write a restart, and exit gracefully if necessary -->
-    <HaltEvent
+    <!--HaltEvent
       name="restarts"
       maxRuntime="$t_allocation$"
-      target="/Outputs/restartOutput"/>
+      target="/Outputs/restartOutput"/-->
```

Comment:
does this need to be commented out?

Reply:
As for my run on Pecan, if this is not commented out, it will cause premature termination of the simulation.